.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish Python client to PyPI
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - v*
```

Comment:
probably we only want to publish a Python package if something in the `python` folder changes when pushed to `main` branch or tagged? Otherwise all `Go` code or even documentation changes will trigger it?

Reply:
Yeah, good point. I'll add a path filter as well.